tests/PilotLive/Models/OpenTableModelTest.php

```diff
@@ -68,4 +68,65 @@ public function testAttributes()
             ],
         ]);
     }
+
+    public function testDetail(){
+        $params = [
+            'reference' => '123abc']
+        ;
```

Comment:
No semi colon on own line - the `]` from the line above should be here (and a trailing comma)